tests/test_vectorstores_pgvector.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
Please update the test scripts name into `test_dataprep_pgvector.sh`

Reply:
After that, the test will be automatically triggered.

Reply:
ok, done